src/Sylius/Bundle/AdminBundle/Resources/config/app/events.yaml

```
@@ -235,6 +235,30 @@ sylius_ui:
                     priority: 5
                     context:
                         event: sylius.admin.layout.topbar_right
+                        
+        sylius.admin.grid.filters:
+            blocks:
+                content:
+                    template: "@SyliusUi/Grid/Filter/_content.html.twig"
+                    priority: 10
+
+        sylius.admin.grid.content:
```

What would you say, that initial event would be called just 
```suggestion
        sylius.grid:
```